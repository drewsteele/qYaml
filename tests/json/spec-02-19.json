{"canonical": 12345, "decimal": "+12,345", "sexagesimal": 12345, "octal": 12, "hexadecimal": 12}