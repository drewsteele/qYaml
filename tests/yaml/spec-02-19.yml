canonical: 12345
decimal: +12,345
# sexagesimal: 3:25:45 - support was removed for sexigesimal in yaml 1.2
octal: 014
hexadecimal: 0xC
