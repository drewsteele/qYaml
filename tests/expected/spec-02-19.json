{"canonical": 12345, "decimal": "+12345", "octal": 12, "hexadecimal": 12}
